src/main/java/Poirot.java

```diff
@@ -0,0 +1,87 @@
+import java.util.Scanner;
+
+public class Poirot {
+    private static Task[] list_actions = new Task[100];
+    private static int last_index = 0;
+
+    public static void echo(String msg) {
+        System.out.println("____________________________________________________________\n");
+        System.out.println(msg);
+        System.out.println("____________________________________________________________\n");
+    }
+
+    public static void print(Task[] list) {
+        if (last_index == 0) {
+            System.out.println("____________________________________________________________\n");
+            System.out.println("No actions available");
+        } else {
+            System.out.println("____________________________________________________________\n");
+            for (int i = 0; i < last_index; i++) {
+                System.out.println((i + 1) + ".[" + list_actions[i].getStatusIcon() + "]" + list_actions[i].getDescription());
```

Comment:
Abstract this to a function and give variable names for list_actions[i].getStatusIcon() and list_actions[i].getDescription to improve code readability